toolchest_client/api/query.py

```diff
@@ -217,6 +217,8 @@ def _register_input_file(self, input_file_path, input_prefix, input_order):
             'input-files'
         ])
         file_name = os.path.basename(input_file_path)
+        if "?" in file_name:
+            file_name = file_name.split("?")[0]
```

Comment:
How does this handle `?`s in local file paths?

This seems specific to HTTP URLs